services/121-service/package.json

```diff
@@ -13,8 +13,8 @@
     "lint": "eslint \"**/*.{ts,js}\"",
     "fix": "npm run lint -- --fix",
     "build": "nest build -p tsconfig.json",
-    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION)  node dist/main.js",
-    "start:dev": "nodemon --watch src/**/*.json --exec npm run start:dev:nest",
+    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION) node --max-old-space-size=4096 dist/main.js",
+    "start:dev": "NODE_OPTIONS='--max-old-space-size=4096' nodemon --watch src/**/*.json --exec npm run start:dev:nest",
```

Comment:
@elwinschmitz Do you see any concerns with increasing the memory size?

Reply:
The [docs on this option](https://nodejs.org/docs/latest-v18.x/api/cli.html#--max-old-space-sizesize-in-megabytes) might help in terms of understanding the recommendations around this.

Reply:
I wonder how much 'room' there is on the Azure-side of this.

Because I'd think the reality in the local Docker-setup will be different from the production-Azure-App-Services setup...

I can't easily spot the existing memory-use... But the whole "App Service Plan" seem to have 16Gb available (shared between all production-instances)...